modules/core/keeper/msg_server.go

```diff
@@ -756,3 +1132,15 @@ func (k Keeper) UpdateConnectionParams(goCtx context.Context, msg *connectiontyp
 
 	return &connectiontypes.MsgUpdateParamsResponse{}, nil
 }
+
+// UpdateChannelParams defines a rpc handler method for MsgUpdateParams.
+func (k Keeper) UpdateChannelParams(goCtx context.Context, msg *channeltypes.MsgUpdateParams) (*channeltypes.MsgUpdateParamsResponse, error) {
```

Comment:
I think we are missing a test?

Reply:
looks like it!

Reply:
Added [here](https://github.com/cosmos/ibc-go/pull/5494).